app/dashboard/static/js/app/view-tests-case-id.2020.3.js

```diff
@@ -0,0 +1,383 @@
+        html.replaceContent(  /* ToDo: link to commit when possible */
+            document.getElementById('git-commit'),
+            document.createTextNode(results.git_commit));
+        html.replaceContent(
+            document.getElementById('arch'),
+            document.createTextNode(results.arch));
+        html.replaceContent(
+            document.getElementById('job-date'), dateNode);
+        html.replaceContent(
+            document.getElementById('status'), ttable.statusNode(status));
+    }
+
+    function updateMeasurements(testCase) {
+        var mesDiv;
+        var mesList;
+
+        mesDiv = document.getElementById("measurements");
+        mesList = document.createElement('dl');
+        mesList.className = 'dl-horizontal';
+        mesDiv.appendChild(mesList);
+
+        testCase.measurements.forEach(function(data) {
+            var unit = document.createElement('dt');
+            var value = document.createElement('dd');
+
+            unit.appendChild(document.createTextNode(data['unit']));
+            mesList.appendChild(unit);
+            value.appendChild(document.createTextNode(data['value']));
+            mesList.appendChild(value);
+        });
+
+        html.removeClass(mesDiv, 'hidden');
+    }
+
+    function updateLogLines(testCase) {
+        var logLines;
+        var logBlock;
+        var logText;
+
+        logLines = document.getElementById('log-lines-content');
+        logBlock = document.createElement('pre');
+        logText = document.createTextNode('');
+        logBlock.appendChild(logText);
+        logLines.appendChild(logBlock);
+
+        testCase.log_lines.forEach(function(line) {
+            logText.data += line.msg + '\n';
+        });
+
+        html.removeClass(document.getElementById('log-lines'), 'hidden');
+    }
+
+    function updateGroupDetails(results) {
+        var translatedURI;
+        var gitNode;
+        var logNode;
+        var defconfigNode;
+        var defconfigPath;
+        var defconfigLink;
+        var buildLink;
+        var kernelNode;
+        var kernelPath;
+        var modulesNode;
+        var dtbNode;
+
+        translatedURI = urls.createFileServerURL(gFileServer, results);
+
+        gitNode = document.createElement('a');
+        gitNode.appendChild(document.createTextNode(results.git_url));
+        gitNode.href = results.git_url;
+        gitNode.title = "Git URL";
+
+        logNode = tcommon.logsNode(
+            results.boot_log, results.boot_log_html, results.lab_name,
+            translatedURI[0], translatedURI[1]);
+
+        defconfigNode = html.tooltip();
+        defconfigLink = document.createElement('a');
+        defconfigLink.appendChild(
+            document.createTextNode(results.defconfig_full));
+        defconfigPath = translatedURI[1] + "/kernel.config";
+        defconfigLink.href =
+            translatedURI[0].path(defconfigPath).normalizePath().href()
```

Comment:
It works as far as I can tell, see this page for example:
https://staging.kernelci.org/test/case/id/5e94a5b59c954596db219e58/

Can you please provide a link to a page where it doesn't work?

Reply:
take a look on this page: https://staging.kernelci.org/test/case/id/5e963b0edb5c1a46d9b8e8f9/ 
 
I saw that it works for most cases, but..

Reply:
Great, thanks for spotting this.  It looks like other test results with the same build are fine, so that's a strange corner-case.  I'll take a look tomorrow.

Reply:
Found the root cause, it's an issue with sub-groups missing some values.  Here's a PR to fix it in the backend:
https://github.com/kernelci/kernelci-backend/pull/232

We'll see in tomorrow's staging data if this is resolved.  It worked with my local set-up.

Reply:
There was one more issue with the`modules_dir`, this is actually not used anywhere so I've updated the code to look for `modules` only instead.  That worked:
https://staging.kernelci.org/test/case/id/5e97e876e1e1cb6b7888cbb0/